### DISCUSSION TITLE
How to store logs directly to a database?

### DISCUSSION THREAD

**Discussion by shahidcodes** [0xERR0R/blocky#275] [Q&A] [answered]
Feature list says that blocky can store logs to database but I didn't find anything in docs how to do that?

**Comment by 0xERR0R** [comment, marked as answer]
This feature is on the development branch and was not released yet. You can test it, just use docker image with development tag or compile the development branch.

Configuration example https://github.com/0xERR0R/blocky/blob/development/docs/config.yml
